miner/worker.go

```diff
@@ -579,8 +535,120 @@ func (w *worker) updateSnapshot() {
 	w.snapshotState = w.current.state.Copy()
 }
 
+func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Address) (error, []*types.Log) {
```

Comment:
Please swap the returns to `([]*types.Log, error)`, not sure how this error ended up as the first return value.